web/src/lib/components/photos-page/asset-grid.svelte

```diff
@@ -296,7 +297,9 @@
 <svelte:window on:keydown={onKeyDown} on:keyup={onKeyUp} on:selectstart={onSelectStart} />
 
 {#if showShortcuts}
-  <ShowShortcuts on:close={() => (showShortcuts = !showShortcuts)} />
+  <FullScreenModal on:clickOutside={() => (showShortcuts = !showShortcuts)}>
```

Comment:
Can you move the fullscreen modal to inside `ShowShortcuts`? `clickOutside` should `dispatch('close')` from within the component.